### PR TITLE
fixed the styling for all the text boxes in various cards that tend t…

### PR DESCRIPTION
**Pull request by vikas199**
…o break

### Jira Story

- [Add current location of child to incident informaton card HOT-1881](https://osi-cwds.atlassian.net/browse/HOT-1881)

## Description
This story is about bug fixing the length of the text area box of 256 characters in incident information form and alignment of the text on the show mode so that it does not cross the card. There is also a slight refactor added to the address types.

## Tests
- [ ] I have included unit tests 
- [ ] I have included feature tests 
- [ ] I have included other tests 
- [x] I have NOT included tests 

## Types of changes

- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (No behavioral changes)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [x] I have ran all tests for the project.
- [x] I have ran lint/rubocop check for the changed/new files.
- [x] My code is in a stable state ready to be deployable, but not necessarily complete.
- [x] I promise on my honor as a CWDS developer to ensure I don't break the build, to check Lint/Rubocop, use best practices, to leave the code in better shape than I found it, and to have a good day.

### CODE REVIEW

**Comment by vikas199** [comment]
jenkins retest this please

File: app/javascript/views/NarrativeShow.jsx [hunk 1]
```diff
@@ -5,7 +5,7 @@ import ShowField from 'common/ShowField'
 const NarrativeShow = ({narrative, errors}) => (
   <div className='card-body'>
     <div className='row'>
-      <ShowField gridClassName='col-md-12' label='Report Narrative' errors={errors} required>
+      <ShowField gridClassName='col-md-12 text-area-show' label='Report Narrative' errors={errors} required>
```
Comment (wmitchell-cambria) [inline app/javascript/views/NarrativeShow.jsx]: My problem with this is that `text-area-show` is not a `gridClassName`. `gridClassName` refers to Bootstrap classnames like `col-md-12`, and by the name of it, it only expects a single one. Passing multiple classes to it seems like it is asking for trouble, especially when those are not Bootstrap grid classes.

There are a few ways you could go about this:
* If this should apply to all `FormFields`, I would include it within the `FormField` component rather than setting it manually each place `ShowField` is used.
* If it should only apply to some FormFields, I would consider splitting it into a new component that clearly identifies when and where it should be used. Creating a component like `BlockShowField` would indicate that these fields support wrapping block text, and then the class could be encapsulated within that component.
* If we really don't want to create a new component for this, we could add a new prop (or rename the `gridClassName` prop) to more clearly indicate that we expect it to handle a string of arbitrary classNames.

File: app/javascript/views/IncidentInformationForm.jsx [hunk 2]
```diff
@@ -100,7 +100,7 @@ const IncidentInformationForm = ({incidentDate, errors, onChange, onBlur, addres
           id='current_location_of_children'
           onChange={({target: {value}}) => onChange(['current_location_of_children'], value)}
           value={locationOfChildren}
-          maxLength='250'
+          maxLength='256'
```
Comment (wmitchell-cambria) [inline app/javascript/views/IncidentInformationForm.jsx]: FWIW, this change is unrelated to the PR topic, and from the story, it sounds like it is actually the most important piece to merge. Separating this out into its own PR (or its own commit) would make it easier to move this forward if the other changes become blocked.

File: app/javascript/selectors/screening/personShowSelectors.js [hunk 3]
```diff
@@ -186,7 +185,7 @@ export const getAllPersonFormattedAddressesSelector = (state, personId) => (
         state: formattedState(address.get('state')),
         zip: address.get('zip'),
         zipError: address.get('legacy_id') ? null : getZIPErrors(address.get('zip')),
-        type: systemCodeDisplayValue(address.get('type'), getAddressTypes(state)) || address.get('type'),
```
Comment (wmitchell-cambria) [inline app/javascript/selectors/screening/personShowSelectors.js]: Same thing here. These changes regarding address types are unrelated to the topic of the PR, and it seems like a pretty important bug to merge in. Separating it into its own commit provides flexibility in what we merge.

File: app/javascript/views/IncidentInformationForm.jsx (hunk 2 above)
Comment (vikas199) [reply app/javascript/views/IncidentInformationForm.jsx]: This PR is about character limit. But thought of making some refactor into the code instead of creating a separate PR
